packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/errors.py

```diff
@@ -0,0 +1,13 @@
+from pydantic.errors import PydanticErrorMixin
+
+
+class BaseDynamicSchedulerRPCError(Exception, PydanticErrorMixin):
+    ...
+
+
+class ServiceWaitingForManualInterventionError(BaseDynamicSchedulerRPCError):
+    msg = "service waiting for manual intervention"
+
+
+class ServiceWasNotFoundError(BaseDynamicSchedulerRPCError):
+    msg = "service waiting for manual intervention"
```

Comment:
This message is identical to the one above. Should you perhaps make a distintion between these two error messages?